cpp/oneapi/dal/common.hpp

```diff
@@ -96,6 +96,62 @@ using v1::base;
 using v1::data_type;
 using v1::range;
 
+class result_option_id_base {
+    using this_t = result_option_id_base;
+    using bitset_t = std::uint64_t;
+
+public:
+    result_option_id_base() = default;
```

Comment:
Uninitialized `mask_` member variable.

Reply:
In such case smth like `if(input_result_options & result_options::res_opt1)` would be illegal. And to be honest I don't see any other reason to add explit keyword

Reply:
Fixed